### DISCUSSION TITLE
the lib is not work with laravel 10

### DISCUSSION THREAD

**Discussion by hanbz** [spatie/laravel-google-cloud-storage#83] [Q&A]
I used Laravel 10 to build an API for file uploads. I've completed the setup and installation, including configuring using GOOGLE_CLOUD_KEY_FILE. The file has been verified and there seem to be no issues. However, when using the following code to upload the file:

Storage::disk('gcs')->put('books', $request->file('avatar'));

The file upload isn't successful, and there are no exceptions or errors being thrown.

**Comment by mxlauc** [comment]
Hello

I just tried my project made in Linux, cloned in Windows, and I just faced this same problem. Storage::disk('gcs')->put() returned false, the file was not uploaded, and no error message was displayed.

Trying other libraries (that are not related to the topic) I saw messages like:
_curl: (60) SSL certificate problem: unable to get local issuer certificate._

I assumed that this message was also causing my first problem, so I followed the steps in this link:
https://stackoverflow.com/questions/24611640/curl-60-ssl-certificate-problem-unable-to-get-local-issuer-certificate

Now Storage::disk('gcs')->put() returns true and the files upload fine.

**Reply by mxlauc** [reply]
By the way, to display the error messages it is necessary to activate this option

```
'gcs' => [
            'driver' => 'gcs',
            'throw' => true,
],
```
